api/api.py

```diff
@@ -199,44 +228,39 @@ class CourtModel(BaseModel):
 
     court_name: str
 
-    class Config:
-        orm_mode = True
+    model_config = ConfigDict(from_attributes=True)
```

Comment:
Since this is the same for all - if possible, have it either global or as a variable? But I'm not sure if it affects its functionality so maybe not